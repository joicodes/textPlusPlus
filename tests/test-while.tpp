def void start(){
	int x;

	x = 0;
	while(x < 10){
		write("Falalalala lala la la");
		x = x + 1;
	}
}