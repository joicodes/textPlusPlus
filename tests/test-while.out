Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
Falalalala lala la la
 

